app/api-v1/routes/recipe.js

```diff
@@ -5,7 +5,27 @@ module.exports = function (recipeService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
```

Comment:
just to confirm I hope that where we call this controller it's wrapped in `try/catch`